bin/tester/new.js

```diff
@@ -0,0 +1,303 @@
+import path from 'path';
```

Comment:
If this is a refactor can we drop this and put on a branch please

Reply:
ok